### DISCUSSION TITLE
Securly blocked the New-Point-Blank exploit, but i found a fix.

### DISCUSSION THREAD

**Discussion by sexy-potatoes** [3kh0/ext-remover#636] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
they changed the extension ID, so https://tinyurl.com/bettergoofcurly doesn't work.   Here is a link that works:
chrome-extension://joflmkccibkooplaeoinecjbmdebglab/blocked.html?category=PGEgaHJlZj0iYWJvdXQ6YmxhbmsiIHRhcmdldD0iX2JsYW5rIiByZWw9Im9wZW5lciIgc3R5bGU9IiBiYWNrZ3JvdW5kOiB3aGl0ZTsgY29sb3I6IGJsYWNrOyBkaXNwbGF5OiBibG9jazsgaGVpZ2h0OiAzODBweDsgdHJhbnNmb3JtOiBzY2FsZSgyLjUpOyBkaXNwbGF5OiBmbGV4OyBqdXN0aWZ5LWNvbnRlbnQ6IGNlbnRlcjsgYWxpZ24taXRlbXM6IGNlbnRlcjsgIj48ZGl2PkNsaWNrIE1lPC9kaXY%2BIDwvYT4%3D

**Comment by deryilz** [comment]
bud im pretty sure they dont know their extension is vulnerable. they just happened to move it off of the web store
